CampaignConfig/fclless/run_Primary.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+echo "$(date) starting run_Primary"
+echo "args: $@"
+echo "fname=$fname"
+echo "pwd=$PWD"
+echo "ls of default dir"
+ls -al
+IND=$( echo $fname | awk -F. '{print $5}' | sed 's/^0*//' )
+TARF=$(ls $CONDOR_DIR_INPUT/*.tar)
+echo "IND=$IND TARF=$TARF"
+mu2ejobfcl --parfile $TARF --index $IND --default-proto root --default-loc tape > temp.fcl
+echo "$(date) run_Primary temp.fcl content"
```

Comment:
Would a different script be needed for (say) run_Mix.sh?  I don't see anything primary-specific about this script.

Reply:
I think probably not given all the parameters are defined in the parfile. So yes I guess this script is more generic than the name suggests but I havent tested it yet.

This is a first step, it will evolve.